src/app/core/core.module.ts

```diff
@@ -0,0 +1,39 @@
+// modules
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {BrowserModule} from '@angular/platform-browser';
+import {MaterialModule} from './modules/material.module';
+import { RouterModule } from '@angular/router';
+import { AppRoutingModule } from '../app-routing.module';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import { PipesModule } from './pipes/pipes.module';
```

Comment:
In some places you have a spaces between { } and imports, in other places you do not have them. Please use same formatting everywhere

Reply:
fixed